src/main/java/com/epam/ta/reportportal/ws/controller/LaunchController.java

```diff
@@ -208,55 +208,54 @@ public OperationCompletionRS deleteLaunch(@PathVariable String projectName, @Pat
 			@SortFor(Launch.class) Pageable pageable, @AuthenticationPrincipal ReportPortalUser user) {
 		return getLaunchMessageHandler.getProjectLaunches(normalizeId(projectName), filter, pageable, user.getUsername());
 	}
-	//
-	//	
-	//	@RequestMapping(value = "/latest", method = GET)
-	//	@ResponseBody
-	//	@ResponseStatus(OK)
-	//	@ApiOperation("Get list of latest project launches by filter")
-	//	public Page<LaunchResource> getLatestLaunches(@PathVariable String projectName, @FilterFor(Launch.class) Filter filter,
-	//			@SortFor(Launch.class) Pageable pageable) {
-	//		return getLaunchMessageHandler.getLatestLaunches(normalizeId(projectName), filter, pageable);
-	//	}
-	//
-	//	
-	//	@RequestMapping(value = "/mode", method = GET)
-	//	@ResponseBody
-	//	@ResponseStatus(OK)
-	//	//@PreAuthorize(NOT_CUSTOMER)
-	//	@ApiOperation("Get launches of specified project from DEBUG mode")
-	//	public Iterable<LaunchResource> getDebugLaunches(@PathVariable String projectName, @FilterFor(Launch.class) Filter filter,
-	//			@SortFor(Launch.class) Pageable pageable, @AuthenticationPrincipal ReportPortalUser user) {
-	//		return getLaunchMessageHandler.getDebugLaunches(normalizeId(projectName), user, filter, pageable);
-	//	}
 
-	//	@RequestMapping(value = "/tags", method = GET)
-	//	@ResponseBody
-	//	@ResponseStatus(OK)
-	//	@ApiOperation("Get all unique tags of project launches")
-	//	public List<String> getAllTags(@PathVariable String projectName, @RequestParam(value = "filter." + "cnt." + "tags") String value,
-	//			@AuthenticationPrincipal ReportPortalUser user) {
-	//		return getLaunchMessageHandler.getTags(normalizeId(projectName), value);
-	//	}
+	@RequestMapping(value = "/latest", method = GET)
+	@ResponseBody
+	@ResponseStatus(OK)
+	@ApiOperation("Get list of latest project launches by filter")
+	public Page<LaunchResource> getLatestLaunches(@PathVariable String projectName, @FilterFor(Launch.class) Filter filter,
+			@SortFor(Launch.class) Pageable pageable) {
+		return getLaunchMessageHandler.getLatestLaunches(normalizeId(projectName), filter, pageable);
```

Comment:
probably we should replace all normalizeId(projectName) with ProjectUtils.extractProjectDetails() method